### ISSUE TITLE
Routine update of dependencies

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2861]
Update Poetry managed dependencies within existing pyproject.toml constraints.

```
cd /opt/rockstor
poetry update
```

Issue resolution via resulting updated poetry.lock file.

---

Upstream reference: https://python-poetry.org/docs/basic-usage/#updating-dependencies-to-their-latest-versions

**Comment by phillxnet** [comment]
# Resulting Poetry report
Leap 15.6 X86_64 host

```
Package operations: 0 installs, 4 updates, 0 removals

  • Updating importlib-metadata (7.1.0 -> 8.0.0)
  • Updating setuptools (70.0.0 -> 70.2.0)
  • Updating djangorestframework (3.15.1 -> 3.15.2)
  • Updating psycopg (3.1.19 -> 3.2.1)

Writing lock file
```

**Comment by phillxnet** [comment]
# Notable Changelog references:

- djangorestframework: https://www.django-rest-framework.org/community/release-notes/#315x-series
- psycopg: https://www.psycopg.org/psycopg3/docs/news.html

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2862
